atip/utils.py

```diff
@@ -18,7 +18,7 @@ def load_at_lattice(mode="DIAD", **kwargs):
         at.lattice.Lattice: An AT lattice object.
     """
     filepath = os.path.join(
-        os.path.dirname(__file__), "".join(["rings/", mode.lower(), ".mat"])
+        os.path.dirname(__file__), "".join(["rings/", mode, ".mat"])
```

Comment:
I'm fairly certain these matlab files are automatically generated from another script.

Are you basing this on changes to those scripts?

Reply:
The script that generates the `.mat` files saves them using whatever filename the user passes and so is case-sensitive.
This change was made so that ATIP is now also case-sensitive to mode/`.mat` file names.

This originally arose because @JoshuaAppleby added a new i04 lattice and ATIP couldn't load the corresponding `.mat` file because the filename was a different case to all the existing `.mat` files in ATIP.